godo.go

```diff
@@ -294,6 +298,15 @@ func SetRequestHeaders(headers map[string]string) ClientOpt {
 	}
 }
 
+// SetRateLimit sets an optional rate limiter that restricts the number of
+// queries per second that the client can send to enforce QOS.
+func SetRateLimit(rps float64) ClientOpt {
```

Comment:
Might be worth renaming this function to make it more clear that it's constant rate limit.

Reply:
:+1:  Went with`SetStaticRateLimit`.